3rdpartylibrary/worldwind-release/build.gradle

```diff
@@ -1,17 +0,0 @@
-apply plugin: 'nebula.maven-base-publish'
-apply plugin: 'nebula.maven-nebula-publish'
-
-configurations.maybeCreate("default")
-artifacts.add("default", file('worldwind-release.aar'))
-
-// worldwind-release.aar was downloaded form https://github.com/NASAWorldWind/WorldWindAndroid/releases/
-
-publishing.publications {
-    nebula(MavenPublication) {
-        artifactId = "worldwind-release"
-
-        artifacts = [
-            "./worldwind-release.aar" // commit id = b242787
-        ]
-    }
-}
```

Comment:
Is this file not used at all?

Reply:
The only thing in this module was the build.gradle file and it builds fine without it. Can put it back in if it's in use @rsupnekar

Reply:
@rsupnekar is the worldwind-release build ever used?  do we publish it anywhere?

Reply:
This was used before we got versioned releases of NASA WW.  I guess I missed the cleanup when I put WW dependency directly in our gradle files.  Yes it is OK to remove.